packages/test/test-end-to-end-tests/src/test/blobsisAttached.spec.ts

```diff
@@ -65,6 +66,48 @@ describeNoCompat("blob handle isAttached", (getTestObjectProvider) => {
 			provider.updateDocumentId(container.resolvedUrl);
 		});
 
+		it("blob is aborted before uploading", async function () {
```

Comment:
do we have unit test for blob manager? would be good to test at that layer if we can. fine to have some e2e, but unit tests are faster to run and easier to debug

Reply:
should I write tests there covering the same scenarios I have here? It seems a little repetitive to me but maybe I'm not seeing different scenarios that would be covered in blobmanager tests

Reply:
I'm trying to add similar scenarios for blobmanager tests but it seems like the test is not catching the abort error